task-launcher/src/tasks/hearts-and-flowers/timeline.js

```diff
@@ -185,11 +184,11 @@ function getHeartOrFlowerInstructionsSection(adminConfig, stimulusType) {
   // Instruction practice trials do not advance until user gets it right
   subtimeline.push({
     timeline: [instructionPractice1, instructionPracticeFeedback],
-    loop_function: (data) => store.session.get('correct') === false,
+    loop_function: (data) => hfStore().correct === false,
```

Comment:
Is there a reason you have a specific store for hearts and flowers. If in the future we want to put this inside our shared tasks, this will cause even more diversion?

Reply:
I was going back and forth on this. My thought was to keep the taskStore from getting bloated. Although we don't have much state so maybe it makes more sense to keep it all together.

Reply:
Yeah that way when and if we merge all tasks together, all the store logic is in one place. With typescript, we can further tighten it down in the future.